src/components/AvatarWithImagePicker.js

```diff
@@ -52,6 +54,9 @@ const propTypes = {
     /** Size of Indicator */
     size: PropTypes.string,
```

Comment:
Could you please add the correct type here? 
```suggestion
    size: size: PropTypes.oneOf([CONST.AVATAR_SIZE.LARGE, CONST.AVATAR_SIZE.DEFAULT]),
```